classes/service/discussion_service.php

```diff
@@ -81,10 +81,12 @@ public function handle_add_discussion($course, $cm, $forum, $context, array $opt
                     $groupstopostto[] = $groupid;
                 }
             }
-        } else {
+        } elseif (isset($_POST['groupinfo'])) {
```

Comment:
Is this right? Based on the param definition in the edit_controller, groupid is not an interable?

Reply:
we did a couple test on my local, the groupinfo passed the correct id's and added discussions per selected ids